app/assets/javascripts/pretty-text/engines/discourse-markdown/upload-protocol.js.es6

```diff
@@ -4,7 +4,6 @@ function addImage(uploads, token) {
     for (let i = 0; i < token.attrs.length; i++) {
       if (token.attrs[i][1].indexOf("upload://") === 0) {
         uploads.push([token, i]);
-        break;
```

Comment:
hmmm why was this break in there?

Reply:
I believe it's not possible for the `upload://` to be present more than once. I think I removed it by accident, will add it again.